From 3a53a64bd5f49c3c6db7a42fca92142896ecc19d Mon Sep 17 00:00:00 2001
From: Moritz Schulte <mo@g10code.com>
Date: Mon, 16 Feb 2009 01:24:18 +0000
Subject: [PATCH 07/52] 2009-02-16  Moritz  <moritz@gnu.org>

	* poldi.texi (Configuration): Documentation bug fix: the option is
	named "log-file", not "logfile". Thanks, Malte.
---
 doc/ChangeLog  | 5 +++++
 doc/poldi.texi | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/doc/ChangeLog b/doc/ChangeLog
index 42e3991..a98d52f 100644
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,3 +1,8 @@
+2009-02-16  Moritz  <moritz@gnu.org>
+
+	* poldi.texi (Configuration): Documentation bug fix: the option is
+	named "log-file", not "logfile". Thanks, Malte.
+
 2008-12-22  Moritz  <moritz@gnu.org>
 
 	* poldi.texi (Overview): Use footnotes plus cosmetic change.
diff --git a/doc/poldi.texi b/doc/poldi.texi
index 2b20096..cd0fa35 100644
--- a/doc/poldi.texi
+++ b/doc/poldi.texi
@@ -231,7 +231,7 @@ the PAM configuration files as arguments to the Poldi PAM module (with
 standard doubledash notation).
 
 @table @code
-@item logfile FILENAME
+@item log-file FILENAME
 Specify the file to use for log messages.
 @item auth-method AUTH-METHOD
 Specify the authentication method to use.  May be either ``localdb
-- 
2.17.0

